src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -307,16 +308,16 @@ public boolean isUseLLVM() {
                 .orElse(Collections.emptyList());
     }
 
-    public List<String> getDelayInitList() {
-        return delayInitList;
+    public List<String> getInitBuildTimeList() {
+        return initBuildTimeList;
     }
 
     /**
      * Sets additional lists
-     * @param delayInitList a list of classes that will be added to the default delayed list
+     * @param initBuildTimeList a list of classes that will be added to the default delayed list
```

Comment:
that will be added to the initialize-at-buildtime list (as opposed to delayed)

Reply:
Fixed